rust/src/tests/chown.rs

```diff
@@ -2,17 +2,17 @@ use nix::unistd::chown;
 
 use crate::{runner::context::TestContext, utils::lchown};
 
-use super::errors::enotdir::assert_enotdir_comp;
+use super::errors::enotdir::enotdir_comp_test_case;
 
-assert_enotdir_comp!(chown, |ctx: &mut TestContext, path| {
+enotdir_comp_test_case!(chown, |ctx: &mut TestContext, path| {
     let user = ctx.get_new_user();
     chown(path, Some(user.uid), None)
 });
 
 mod lchown {
     use super::*;
 
-    assert_enotdir_comp!(lchown, |ctx: &mut TestContext, path| {
+    enotdir_comp_test_case!(lchown, |ctx: &mut TestContext, path| {
```

Comment:
I'm not sure if it's better to separate the lchown/lchmod in other modules since we test them in the same test functions before. We could make the macro take multiple functions to test them in the same test function. What do you think about it?

Reply:
Better to leave them separate.  Combining them into the same function, as you do in PR #108 , isn't great.  In that case I think it's OK for now because tests/chown/00.t is so complicated that it would be hard to separate them at the same time that you convert them.  Ideally, we would separate them later.  But here, separation is better, as usual.